public_dropin_environments/r_lang/Dockerfile

```diff
@@ -2,13 +2,22 @@
 # It contains a variety of common useful data-science packages and tools.
 FROM datarobot/r-dropin-env-base
 
+RUN apt update -y && apt install -y software-properties-common && \
+    add-apt-repository -y ppa:deadsnakes/ppa && \
+    apt update -y && apt install -y python3.7 python3.7-dev
+
+RUN python3.7 -m pip install setuptools wheel virtualenv && \
```

Comment:
Can't you just use `python3 -m ...`? (instead of `python3.7 -m ...`)

Reply:
At this stage default Py3.6 is still default...
So I first need to install virtualenv,  and then work in virtual env.

I consider this a temp solution, as we'll need to bump distro, python and R  versions for R env.

Reply:
why do we need 3.7?